src/elements/content-uploader/ItemAction.js

```diff
@@ -101,7 +101,12 @@ const ItemAction = ({
         <div className="bcu-item-action">
             {tooltip ? (
                 <Tooltip position="top-left" text={intl.formatMessage(tooltip)}>
-                    <PlainButton onClick={onClick} type="button" isDisabled={status === STATUS_STAGED}>
+                    <PlainButton
+                        aria-label={intl.formatMessage(tooltip)}
+                        isDisabled={status === STATUS_STAGED}
```

Comment:
super minor and not absolutely necessary! But for readability, should we put this in a variable rather than inline it? (haven't seen this inlining pattern for props that often elsewhere)

Reply:
I'm all for making things more readable! I only see my perspective so it's helpful to hear where things can be changed :D Just to make sure, do you mean aria-label should be defined as a variable or this `status === STATUS_STAGED`?

Reply:
I think I did both with @tjuanitas 's help!

Reply:
Ah I meant the `status === STATUS_STAGED`, but it sounds like you've got it! :)